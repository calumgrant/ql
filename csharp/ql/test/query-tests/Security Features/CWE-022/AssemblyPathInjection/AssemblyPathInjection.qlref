Security Features/CWE-022/AssemblyPathInjection.ql