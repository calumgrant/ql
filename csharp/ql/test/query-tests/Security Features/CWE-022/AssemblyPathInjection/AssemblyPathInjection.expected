| Test.cs:14:40:14:50 | access to local variable libraryName | $@ flows to here and is used as the path to load a DLL. | Test.cs:11:30:11:52 | access to property QueryString | User-provided value |
